samples/universal-react-node/config/default.json

```diff
@@ -1,14 +1,9 @@
 {
   "plugins": {
-    "electrode-csrf-jwt": {
-      "options": {
-        "secret": "shhhhhh",
-        "expiresIn": 60
-      }
-    },
```

Comment:
fetch needs to pass `x-csrf-jwt` https://github.com/electrode-io/electrode/blob/master/samples/universal-react-node/client/components/csrf.jsx#L21